transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -16,6 +16,7 @@
 package io.netty.channel;
 
 import io.netty.buffer.ByteBufAllocator;
+import io.netty.channel.socket.DuplexChannel;
```

Comment:
This import shouldn't be added... 

```suggestion
```

Reply:
now cleaning up this and undrafting - @normanmaurer it's ok if I sent a benchmark later?

Reply:
yep